src/hosted-buttons/index.js

```diff
@@ -34,24 +42,59 @@ export const getHostedButtonsComponent = (): HostedButtonsComponent => {
         selector,
       });
 
-      // $FlowFixMe
-      Buttons({
+      const createOrder = buildHostedButtonCreateOrder({
+        enableDPoP,
         hostedButtonId,
-        style,
-        onInit,
+        merchantId,
+      });
+      const onApprove = buildHostedButtonOnApprove({
+        enableDPoP,
+        hostedButtonId,
+        merchantId,
+      });
+
+      const buttonOptions = {
+        createOrder,
+        hostedButtonId,
+        merchantId,
+        onApprove,
         onClick,
-        createOrder: buildHostedButtonCreateOrder({
-          enableDPoP,
-          hostedButtonId,
-          merchantId,
-        }),
-        onApprove: buildHostedButtonOnApprove({
-          enableDPoP,
-          hostedButtonId,
-          merchantId,
-        }),
-      }).render(selector);
+        onInit,
+        style,
+      };
+
+      if (shouldRenderSDKButtons(fundingSources)) {
+        const { flexDirection } = getFlexDirection({ ...style });
+
+        appendButtonContainer({ flexDirection, selector });
+
+        // Only render 2 buttons max
+        fundingSources.slice(0, 2).forEach((fundingSource, index) => {
```

Comment:
You don't have to do this, but imo it might be helpful to leave a comment above this to say it needs to be refactored.  I'm just scared they'll start passing funding sources in and it won't work as expected 😅  

But up to you! If you feel like its unnecessary, I'm down for that as well. 🚀